salt/haproxy/config/haproxy.cfg.jinja

```diff
@@ -65,7 +65,6 @@ defaults
     balance roundrobin
 
     timeout connect 5000
-    timeout client  50000
```

Comment:
is this removed because it is no longer supported in this haproxy version?

Reply:
will investigate within haproxy PR, weekend brain is strong on a monday :)